src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { Filter } from '../../types/Filter';
+import { TodoContext } from '../../context/TodoContext';
+
+interface Props {
+  filter: Filter,
+  setFilter: (newFilter: Filter) => void,
+}
+
+export const TodoFooter: React.FC<Props> = ({
+  filter,
+  setFilter,
+}) => {
+  const {
+    activeTodos,
+    completedTodos,
+    clearCompleted,
+  } = useContext(TodoContext);
+
+  const activeTodosCount = activeTodos.length;
+  const completedTodosCount = completedTodos.length;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodosCount} items left`}
```

Comment:
If there is one item, there should not be a plural in the text.

Reply:
I agree with you, and I changed it, but tests expect other message
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/70105717/a4c69288-c537-4df5-97d0-cf48510a1d1f)